public/ja/faq/index.html

```diff
@@ -574,7 +574,7 @@ <h4 class="panel-title">
 								<div id="question26" class="panel-collapse collapse" style="height: 0px;">
 									<div class="panel-body">
 										<p>
-											Dev protocolは持続かのな活動を支援しているので、将来的に開発する予定がないものに関しては申請が通らない場合があります
+											Dev protocolは持続可能な活動を支援しているので、将来的に開発する予定がないものに関しては申請が通らない場合があります
```

Comment:
This is not related to this change, but `Dev protocol` should be unified to `Dev Protocol`.

Reply:
Thank you for suggesting! 
I'm going to fix it too :)

Reply:
Thanks!! 👍